doc/example_code/how_to_examples/drawing_text_objects.rst

```diff
@@ -2,16 +2,19 @@
 
 .. _drawing_text_objects:
 
-Fast Text Drawing
-=================
+Better Text Drawing with Text Objects
+=====================================
 
 .. image:: drawing_text_objects.png
     :width: 500px
     :align: center
     :alt: Screenshot of drawing with text objects
 
-This example shows how to draw text using "text objects." It is the fastest
-way to draw text. If you are looking for something simpler, see :ref:`drawing_text`.
+This example shows how to draw text using "text objects." It can be many
+times faster than using :py:func:`arcade.draw_text`.
```

Comment:
I would emphasize the performance differences here.

* `draw_text`: 10s of objects per fram
* `Text` object: 100s of objects per frame
* `Text` object with batching: 1000s of objects per frame

The performance difference is crazy. Especially when they are more or less static.

Reply:
For brevity's sake, it seems appropriate to add a programming guide section on text to cover this. We need that anyway to explain issue with fonts.